### PR TITLE
SPCTR-3207 Resolved an issue of Timeline Icons look big in customizer preview.

### PR DESCRIPTION
**Pull request by AbhijitNage123**
### Description

- Timeline Icon look big issue

### Screenshots

- https://d.pr/v/xkVRQP

### Types of changes

- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Go to Astra Customizer
- Select Header builder
- Use widget and drag and drop timeline
- Make sure not big icon issue visible

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [x] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

File: blocks-config/uagb-controls/addInitialAttr.js [hunk 1]
```diff
@@ -37,15 +37,15 @@ const addInitialAttr = ( ChildComponent ) => {
 		]; // Add all parent block name here who's getting issue in customize preview.
 
 		useEffect( () => {
-			if ( uagb_blocks_info.is_customize_preview && undefined === block_id && listOfParentBlock.includes( name ) ) {
+			if ( uagb_blocks_info.is_customize_preview && ( '0' === block_id || undefined === block_id ) && listOfParentBlock.includes( name ) ) {
```
Comment (bsf-zanev) [inline blocks-config/uagb-controls/addInitialAttr.js:40]: Will this work as intended?

```suggestion
			if ( uagb_blocks_info.is_customize_preview && empty( block_id ) && listOfParentBlock.includes( name ) ) {
```

File: blocks-config/uagb-controls/addInitialAttr.js [hunk 2]
```diff
@@ -37,15 +37,15 @@ const addInitialAttr = ( ChildComponent ) => {
...
 				document.addEventListener( `UAG-${name}-${clientId.substr( 0, 8 )}-BlockCustomizeWidgetEditor`, function ( e ) {
 					setAttributes( { block_id: e.detail.id, classMigrate: e.detail.classMigrate, childMigrate: e.detail.childMigrate } );
 				} );
 			}
 		}, [] );
 
 		useEffect( () => {
-			if ( uagb_blocks_info.is_customize_preview && undefined === block_id && listOfParentBlock.includes( name ) ) {
+			if ( uagb_blocks_info.is_customize_preview && ( '0' === block_id || undefined === block_id ) && listOfParentBlock.includes( name ) ) {
```
Comment (bsf-zanev) [inline blocks-config/uagb-controls/addInitialAttr.js:48]: Same here if it works.

File: blocks-config/uagb-controls/addInitialAttr.js (hunk 1 above)
Comment (AbhijitNage123) [reply blocks-config/uagb-controls/addInitialAttr.js:40]: @bsf-zanev Its JS and empty() function not available in JS.

File: blocks-config/uagb-controls/addInitialAttr.js (hunk 2 above)
Comment (AbhijitNage123) [reply blocks-config/uagb-controls/addInitialAttr.js:48]: Not possible due to empty is not a function of JS.

**Review by bsf-zanev — APPROVED** [review]
Looks good to me, it appears we can't shorten the block_id condition with `!! block_id` since it's a string with '0'.

**Review by adi3890 — APPROVED** [review]

**Pull request merged**